rascal-vscode-extension/package.json

```diff
@@ -47,8 +47,22 @@
       {
         "command": "rascalmpl.importModule",
         "title": "Start Rascal Terminal and Import this module"
+      },
+      {
+        "command": "rascalmpl.startDebuggerForRepl",
```

Comment:
Reading this correctly: https://code.visualstudio.com/api/extension-guides/command

It looks like we should only register commands here that are user facing. Like, we don't want this one showing up in the Command palette that a user can execute. Have we tried what happens if we only register the command in the typescript code?

Reply:
You get an error indicating that the TreeViewItem references a command that's not registered. It's possible to prevent the command from showing up in the command palette, though.

Registering the command in the TS code is possible, and the command is then called when the TreeViewItem is clicked. However, we lose the possibility to attach a button icon to the list entries.

I've changed the logic around the command for now.

Reply:
Yeah, my only goal is, I don't want users accidentally triggering that command when they find it in the command list.

Reply:
As the icon is useful for the user, I've reinstated it.